src/app/components/balance.tsx

```diff
@@ -20,5 +20,5 @@ export function Balance(props: BalanceProps) {
     [balances]
   );
 
-  return <Caption>{balance}</Caption>;
+  return <Caption variant="c3">{balance}</Caption>;
```

Comment:
Why this change here?

Reply:
To fit the testnet badge into the app nav in a more visually appealing, I wanted to make that balance text a little smaller.

Original:
<img width="444" alt="image" src="https://user-images.githubusercontent.com/60433268/221995413-c05c3523-245d-402c-a62d-e87646406a2b.png">

Updated:
<img width="444" alt="image" src="https://user-images.githubusercontent.com/60433268/221995369-81b7ba9a-61a9-4a49-9ae3-ed976278d247.png">

Reply:
@mica000 is this ok? @markmhx? Seems small to me.

Reply:
I don't have much of a preference either way. @mica000 wdyt?

Reply:
I can also make it a little bigger to match the STX address:

<img width="441" alt="image" src="https://user-images.githubusercontent.com/60433268/224326321-da1f41df-5504-44b8-9bb2-f0cb24aa9f8c.png">

Reply:
What is the size of font you are using to display the value @mihirKachroo?

Reply:
I use size "c2" from the Caption component in components/typography. It's font size 12 from what I remember.